pkg/apis/scaler/v1alpha2/types_demand.go

```diff
@@ -23,6 +23,9 @@ import (
 // DemandPhase type declares that the value is a Demands phase
 type DemandPhase string
 
+// Zone type declares an availability zone
+type Zone string
```

Comment:
Where is this type used?

Reply:
Changed the demand spec to use this instead of string now